source/robot_model/include/robot_model/Model.hpp

```diff
@@ -517,6 +556,15 @@ inline Model& Model::operator=(const Model& model) {
   return *this;
 }
 
+inline size_t Model::get_number_of_collision_pairs() {
+  return this->geometry_package_paths_.size() ? this->geom_model_.collisionPairs.size() : 0;
```

Comment:
Nit: might be clearer to use `empty()` or do an actual comparison instead of relying on truthy/falsy behaviors.

Reply:
Indeed!